shadows/framework/src/main/java/org/robolectric/shadows/ShadowUserManager.java

```diff
@@ -243,12 +230,12 @@ protected List<UserHandle> getEnabledProfiles() {
   @Implementation(minSdk = R)
   protected List<UserHandle> getAllProfiles() {
     ArrayList<UserHandle> userHandles = new ArrayList<>();
-    if (userManagerState.userProfilesListMap.containsKey(context.getUserId())) {
-      userHandles.addAll(userManagerState.userProfilesListMap.get(context.getUserId()));
+    if (userManagerState.userProfilesListMap.containsKey(getContext().getUserId())) {
```

Comment:
I don't know how costly `reflector()` is, but shouldn't we store `getContext()` in a variable when it is used multiple times in a method?

Reply:
reflector is very fast, it has a lot of caching enabled.

I think these methods are very seldom invoked, UserManager is not very widely used, and is not on the hot path.